recipes/dd-agent.rb

```diff
@@ -135,5 +135,14 @@ def template_vars
   retry_delay 5
 end
 
+# only load system-probe recipe if an agent6 installation comes with it
```

Comment:
I don't think we should do it this way.
First I have to ask: when do we want to ship the system-probe? Automatically on every installs of Agent 6 (except on Windows) or could it be on user intent?
With the addition of the system-probe recipe, all users of the `chef-datadog` cookbook will be able to choose to add it in their runlist/cookbooks/etc. to install the system-probe on their nodes.

Reply:
Good question. We don't want people to do extra hustle to add `system-probe` to their run-list, they don't need to know about this if they don't use it, `dd-agent` recipe is all they need. If they want to use system-probe, just set the `enabled` under `system-probe` to `true` then it should run.

Reply:
I see 👍